### PR TITLE
Add I'm feeling lucky button to public case search by name

### PR DESCRIPTION
**Pull request by paul-codes-1**
Added an "I'm Feeling Lucky" button to Case Search by Name. User will be immediately redirected to first search result if they are feeling lucky.

https://user-images.githubusercontent.com/14105031/178122375-fbda56cc-348c-4b32-af21-0449e17c4040.mov

### CODE REVIEW

**Comment by paul-codes-1** [comment]
@ustaxcourt-deploy plz respond

File: web-client/src/presenter/sequences/Public/submitPublicCaseAdvancedSearchSequence.js [hunk 1]
```diff
@@ -24,3 +27,19 @@ export const submitPublicCaseAdvancedSearchSequence = [
...
+  clearSearchTermAction,
+  validateCaseAdvancedSearchAction,
+  {
+    error: [
+      setAlertErrorAction,
+      setValidationErrorsAction,
+      clearSearchResultsAction,
+    ],
+    success: showProgressSequenceDecorator([
+      clearAlertsAction,
```
Comment (paul-codes-1) [inline web-client/src/presenter/sequences/Public/submitPublicCaseAdvancedSearchSequence.js:41]: can this be removed to make the loading appear smoother?

**Review by tygooch — APPROVED** [review]
This looks good to me can we please get this merged? My client needs this feature ASAP and my manager said under no circumstances will he allow me to use my fork of this repo in prod because apparently it’s a “security” issue when I make changes but not when you guys do 🙄

**Comment by paul-codes-1** [comment]
> 

I'm doing my best, they call it open source, but definitely some government corruption appears to be going on--some contributeours seem to get their PRs bumped to the front of the queue every time 🙄

**Comment by mmarcotte** [comment]
@paulmoliva -- Thank you for your contribution. At this time we are not going to merge this because it is not a feature that our Product Owner is pursuing; though, we have made them aware.

**Comment by paul-codes-1** [comment]
> @paulmoliva -- Thank you for your contribution. At this time we are not going to merge this because it is not a feature that our Product Owner is pursuing; though, we have made them aware.

@mmarcotte ok, a Mr. Gooch may be reaching out to you regarding the appeals process. What are our rights here?

cc @TyGooch

**Comment by tygooch** [comment]
This is a complete atonement. Can I please get admin access to this repository?

**Pull request closed without merging**